src/analysis/data/perf_evmone_pg_marginal_50_1000000.csv

```diff
@@ -0,0 +1,5501 @@
+program_id,sample_id,task_clock,context_switches,page_faults,instructions,branches,branch_misses,L1_dcache_loads,LLC_loads,LLC_load_misses,L1_icache_loads,L1_icache_load_misses,dTLB_loads,dTLB_load_misses,iTLB_loads,iTLB_load_misses
```

Comment:
hm, I think these CSVs are a little too much and won't serve us a lot. Keep it locally at your machine, that's enough.